ansible/roles/installation/nvidia-docker/tasks/main.yml

```diff
@@ -28,3 +28,47 @@
   when: nvidia_docker | bool and not skip_nvidia|bool
   become: yes
   changed_when: false
+
+- name: Check what graphic drivers are used
+  block:
+  - name: install nvidia-prime apt
+    apt:
+      name: nvidia-prime
+      state: present
+    become: yes
+
+  - name: query prime-select
+    command: prime-select query
+    become: yes
+    register: prime_select_results
+
+  - name: switch driver to nvidia
+    command: prime-select nvidia
+    become: yes
+    when: "prime_select_results.stdout != 'nvidia'"
+
+  - name: reboot the computer
+    debug:
+      msg: You used graphic driver has been switched to nvidia. Please reboot your PC and rerun the oneliner.
```

Comment:
Can you think of anything clearer, always struggle with these, I thought it conveyed my point well

Reply:
The graphics card driver was switched to Nvidia. Please reboot your PC and rerun the oneliner.? In fact it might have been only a typo of You -> Your

Reply:
up to you ;)

Reply:
Ah yeah I didn't spot that typo ! Cheers!